autoware_adapi_v1_msgs/planning/msg/VelocityFactor.msg

```diff
@@ -2,14 +2,21 @@
 uint16 UNKNOWN = 0
 
 # constants for status
-uint16 APPROACHING = 1
-uint16 STOPPED = 2
+uint16 APPROACHING = 1  # Vehicle is approaching the factor
+uint16 STOPPED = 2      # Vehicle has stopped due to the factor
 
-# variables
+# fields
+# Pose of the velocity factor in map frame
 geometry_msgs/Pose pose
+# Distance to the velocity factor [m]
 float32 distance
+# Current status (use status constants above)
 uint16 status
+# Behavior type (see PlanningBehavior.msg for valid values)
```

Comment:
Any name can be used to support user extension modules. PlanningBehavior.msg is list of typical behavior names.

Reply:
Fixed. https://github.com/autowarefoundation/autoware_adapi_msgs/pull/107/commits/e6e302b0a3e03d3f3324eb98f7ff0344344c4102